pkg/serverless/aws/arn.go

```diff
@@ -8,11 +8,12 @@ package aws
 import (
 	"encoding/json"
 	"io/ioutil"
+	"path/filepath"
 	"strings"
 	"sync"
 )
 
-const persistedStateFilePath = "/tmp/dd-lambda-extension-cache.json"
+var persistedStateFilePath = filepath.Join("tmp", "dd-lambda-extension-cache.json")
```

Comment:
`tmp` folder does not exist on windows. You need to use https://golang.org/pkg/io/ioutil/#TempDir or similar function to get a temporary folder.

Reply:
🙇‍♂️